### PR TITLE
chore(deps): Upgrade FEC dependencies, clean unused ones

### PR DESCRIPTION
**Pull request by gkarat**
Fixes https://issues.redhat.com/browse/RHINENG-12005.

# Description

Associated Jira ticket: https://issues.redhat.com/browse/RHINENG-12005.

This bumps the version of FEC/config package. Also, the PR removes unused babel dependencies and cleans up fec.config.js.

# How to test the PR

The application should work without any changes.

# Checklist:

- [x] The commit message has the Jira ticket linked
- [x] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,19 +77,13 @@
...
+    "@redhat-cloud-services/frontend-components-config": "^6.3.1",
     "@redhat-cloud-services/tsc-transform-imports": "^1.0.7",
     "@testing-library/jest-dom": "^6.2.0",
     "@testing-library/react": "^14.1.2",
     "@testing-library/react-hooks": "^7.0.1",
     "@testing-library/user-event": "^14.5.2",
-    "babel-core": "^7.0.0-bridge.0",
     "babel-loader": "^8.2.2",
-    "babel-plugin-dual-import": "^1.2.1",
-    "babel-plugin-istanbul": "^6.1.1",
```
Comment (bastilian) [inline package.json:91]: This is still needed for coverage collection in `cypress`.

File: package.json [hunk 2]
```diff
@@ -111,9 +103,7 @@
     "stylelint-scss": "^3.19.0",
     "ts-patch": "^3.1.2",
     "typescript": "^5.3.3",
-    "webpack": "^5.44.0",
-    "webpack-bundle-analyzer": "^4.4.2",
-    "webpack-cli": "^4.7.2"
+    "webpack-cli": "^5.1.4"
```
Comment (bastilian) [inline package.json:106]: It should be possible to also remove this webpack package.

**Comment by gkarat** [comment]
Added @cypress/webpack-dev-server as a dependency since it didn't allow to spin up cypress tests

**Review by adonispuente — APPROVED** [review]
LGTM, app also behaved as normal when tested.

**Pull request merged**